### PR TITLE
Fix the Makefile to work with the install instructions in the README

### PR DESCRIPTION
**Pull request by Bobb-E-Burns**
**Before submitting a pull request,** please make sure the following is done:

- [x] Fork [the repository](https://github.com/todotxt/todo.txt-cli) and create your branch from `master`.
- [ ] If you've added code that should be tested, add tests!
- [ ] Ensure the test suite passes.
- [ ] Format your code with [ShellCheck](https://www.shellcheck.net/).
- [x] Include a human-readable description of what the pull request is trying to accomplish.
- [ ] Steps for the reviewer(s) on how they can manually QA the changes.
- [ ] Have a `fixes #XX` reference to the issue that this pull request fixes.

First time working with a Makefile but all I did was fix it so the instructions in the README actually work.  
`INSTALL_DIR, CONFIG_DIR & BASH_COMPLETION` were not used in the original Makefile, instead there were `bindir, sysconfdir & datarootdir`.  
Also removed all the references to `DESTDIR` because that was not used at all and did nothing.

The default install location is still the same as I've changed nothing besides naming.

### CODE REVIEW

**Comment by iamleot** [comment]
Hello LeLobster,
just a comment regarding DESTDIR!

LeLobster writes:
> [...]
> Also removed all the references to `DESTDIR` because that was not used at all and did nothing.
> [...]

This is often used in order to install software in a staging directory
and in some package systems all packages actually need to support
DESTDIR.  I think it is better to keep it.

**Comment by Bobb-E-Burns** [comment]
I did not know that.  
No problem, I'll move them back right away.

EDIT -> No idea why I named it `DEST_DIR` in the commit message but it's fine in the Makefile

**Comment by Bobb-E-Burns** [comment]
On second thought, after I did some reading about the Makefile standards on `https://www.gnu.org/prep/standards/html_node/Makefile-Conventions.html` which I probably should've done before submitting a PR, let's ignore this one and keep the original names.

How about just adding a check if those dirs are passed on the command line and assign them to `bindir, sysconfdir & datarootdir` if they are?

File: Makefile [hunk 1]
```diff
@@ -10,13 +10,28 @@ INSTALL_DATA = $(INSTALL) -m 644
...
-sysconfdir = $(prefix)/etc
-
-datarootdir = $(prefix)/share
+ifdef CONFIG_DIR
+	sysconfdir = $(CONFIG_DIR)
+else
+	sysconfdir = $(prefix)/etc
+endif
+
+ifdef BASH_COMPLETION
```
Comment (yuvalif) [inline Makefile:30]: the readme has both BASH_COMPLETION and BASH_COMPLETION_DIR. should probably be fixed there as well

**Comment by karbassi** [comment]
Thoughts @inkarkat?

It looks good to me at first glance.

File: Makefile [hunk 2]
```diff
@@ -10,13 +10,28 @@ INSTALL_DATA = $(INSTALL) -m 644
...
+	bindir = $(INSTALL_DIR)
+else
+	bindir = $(prefix)/bin
+endif
 
 # The directory to install the config file in.
-CONFIG_DIR = $(prefix)/etc
-
-BASH_COMPLETION = $(prefix)/share
+ifdef CONFIG_DIR
```
Comment (iamleot) [inline Makefile]: a

**Review by inkarkat — APPROVED** [review]
Looks fine to me.

**Pull request merged**